README.md

```diff
@@ -239,3 +240,66 @@ Builder.register(AvatarCell, "User-AvatarCell");
 ```
 
 When in doubt, the cell name corresponds to the `data-cell` attribute on the DOM element.
+
+
+## Webpack configuration
+
+If you use webpack, you can easily configure it.
+
+#### 1. Import the context of the folder in which you have cells. Next import the library into app.js, which is a webpack input file:
+
+```js
+// assets/src/js/app.js
+
+...
+const importAll = function(r) {
+  r.keys().forEach(r)
+}
+
+importAll(require.context("../../../lib/YOUR_DIRECTORY/cells", true, /\.js$/))
+
+import Builder from "@vendor/cell/builder" // <-- @vendor is an alias from webpack.config.js, take a look below.
+...
+```
+
+#### 2. Create aliases in the webpack configuration file:
+
+```js
+// assets/webpack.config.js
+
+...
+  resolve: {
+    symlinks: false,
+    modules: ["node_modules"],
+    alias: {
+      "@": path.resolve(__dirname, "./src/js"),
+      "@cells": path.resolve(__dirname, "../lib/YOUR_DIRECTORY/cells"),
+      "@vendor": path.resolve(__dirname, "./vendor"),
+      "@libs": path.resolve(__dirname, "./node_modules")
+    }
+  },
+...
+```
+
+#### 3. Also works fine with [cell-js](https://github.com/DefactoSoftware/cell-js) plugin. Just create the index.js file in the target cell and use the aliases to import the library.
+
+#### E.g. calendar:
+
+```js
+// cells/calendar/index.js
+
+...
+import Cell  from "@vendor/cell/cell"
+import Builder from "@vendor/cell/builder"
```

Comment:
How did you install cell-js? I would expect this:

```js
import { Cell, Builder } from "@defacto/cell-js";
```

Would you mind sharing your webpack `entry` config?

Reply:
Thanks, I'll respond for couple hours :)

Reply:
In assets i have src folder and app.js file which is the entry point of my webpack.config.js

```js
  entry: {
    "./js/app.js": "./src/js/app.js"
  },
```
But now i see that i have a copy of builder.js and cell.js lib in vendor folder. I'll try import it from node modules.

Reply:
> But now i see that i have a copy of builder.js and cell.js lib in vendor folder. I'll try import it from node modules.

That explains 😄
Let me know if importing the package works fo you. If so I'd say we use this in the examples so we can get this merged 🚀